File: atcoder/piano.cpp
#include <iostream>
using namespace std;

void solve() {
    int w, b;
    cin >> w >> b;
    if(w % 6 == 0 and b % 6 == 0)
        cout << "Yes" << endl;
    else
        cout << "No" << endl;

}


int main() {
    int t = 1;
    // cin >> t;
    while(t--) {
        solve();
    }
    return 0;
}

File: atcoder/piano.py
def solve():
    w, b = input().split(' ')
    w = int(w)
    b = int(b)
    pattern = 'wbwbwwbwbwbw'
    pattern *= max(w, b)
    target = 'w' * w + 'b' * b
    if target in pattern:
        print('Yes')
    else:
        print('No')

solve()
